### PR TITLE
front: move NGE logic to dedicated component directory

### PR DESCRIPTION
**Pull request by emersion**
Move low-level NGE details to a separate directory to make sure it stays clearly split from the OSRD-specific logic.

### CODE REVIEW

**Review by kmer2016 — APPROVED** [review]
LGTM

File: front/src/applications/operationalStudies/views/v2/ScenarioV2.tsx [hunk 1]
```diff
@@ -7,14 +7,11 @@ import { GiElectric } from 'react-icons/gi';
...
 import BreadCrumbs from 'applications/operationalStudies/components/BreadCrumbs';
-import NGE from 'applications/operationalStudies/components/MacroEditor/NGE';
 import handleOperation from 'applications/operationalStudies/components/MacroEditor/ngeToOsrd';
 import importTimetableToNGE from 'applications/operationalStudies/components/MacroEditor/osrdToNge';
-import type {
-  NetzgrafikDto,
-  NGEEvent,
-} from 'applications/operationalStudies/components/MacroEditor/types';
 import MicroMacroSwitch from 'applications/operationalStudies/components/MicroMacroSwitch';
+import NGE from 'applications/operationalStudies/components/NGE/NGE';
```
Comment (theocrsb) [inline front/src/applications/operationalStudies/views/v2/ScenarioV2.tsx:13]: Perhaps the file name should be changed to avoid having `/NGE/NGE`

Comment (emersion) [reply front/src/applications/operationalStudies/views/v2/ScenarioV2.tsx:13]: Yup, I agree that would be nicer! Though, everywhere else in OSRD we always duplicate the directory name and the filename. So I'd argue this is something we ought to change throughout the whole codebase (I would rather keep everything consistent).

**Review by Uriel-Sautron — APPROVED** [review]
LGTM !!! Tested.

**Review by theocrsb — APPROVED** [review]
LGTM

**Pull request merged**